### PR TITLE
Added version information issue152

### LINKED ISSUE #152: Versioning: R5.1 update
The initial commit for a new release should be an update to the version information (see example at https://github.com/Project-Counter/cop5/commit/49b75cbed1eabad7afe13d755242e7a81d3de49a) 

The timeline for 5.1 in the README.rst also needs to be updated.

### PR DESCRIPTION
**Pull request by tashalouizamc**
Includes information (release number, release date, compliance date) for R5.0.3 and R5.1

### CODE REVIEW

**Comment by tashalouizamc** [comment]
Closes https://github.com/Project-Counter/cop5/issues/152

File: README.rst [hunk 1]
```diff
@@ -3,9 +3,13 @@ COUNTER Code of Practice Release 5
 
 This repository contains the source for the `COUNTER <https://www.projectcounter.org/>`_ Code of Practice Release 5.
 
-There are three branches:
+There are multiple branches:
 
-* `5.0.2 <https://github.com/Project-Counter/cop5/tree/5.0.2>`_: Release 5.0.2 (published 28 September 2021) will become the current Code of Practice and the requirement for COUNTER compliance effective from January 2022.
+* `5.1 <https://github.com/Project-Counter/cop5/tree/5.0.3>`_: Release 5.1 (published April 2023) will become the current Code of Practice and the requirement for COUNTER compliance effective from January 2025.
+
+* `5.0.3 <https://github.com/Project-Counter/cop5/tree/5.0.3>`_: Release 5.0.3 (published April 2023) will become the current Code of Practice and the requirement for COUNTER compliance effective from January 2024.
```
Comment (berndoberknapp) [inline README.rst:10]: 5.0.3 is a bugfix only release, therefore it should become effective when it is published.

**Comment by berndoberknapp** [comment]
5.0.3 is a bugfix only release that should become effective when it is published.

**Review by tashalouizamc — COMMENTED** [review]
Amended 5.0.3 compliance to immediate

**Pull request merged**